### ISSUE TITLE
MonoDevelop Update

### ISSUE THREAD

**Issue by HUGEwindow** [dkucinskas/MonoDevelop-Styles#1]
Does this theme work with the latest build of MonoDevelop?

I am unable to select the xml in 'add'  in 'syntax highlighting'

**Comment by dkucinskas** [comment]
I don't think so.  At least in Xamarin Studio they changed theme file format.
